### ISSUE TITLE
It is hard to find version 3 ignore attribute in google

### ISSUE THREAD

**Issue by DerekSMorin** [nunit/docs#20]
If you google "nunit ignore attribute" the top results are all for earlier versions.

**Comment by ChrisMaddock** [comment]
I absolutely agree - I used to hit this problem all the time, I find I have to search straight on the wiki here to get anything useful. It additionally doesn't seem to help that the wiki was moved repositories - if google does have a v3 link, it's outdated.

Something like a [canonical link](http://stackoverflow.com/q/29603839/1768779) solution seems the way to go. I don't believe we could implement such a solution in the GitHub wiki however. (Although it may be useful for the old v2 docs site - searching for "nunit ignore attribute" currently links me to v2.4.8)

The new nunit.github.io website - is that a work in progress? Having the docs on a page there, backed by a repository would give us control over things like this.

**Comment by CharliePoole** [comment]
@DerekSMorin As a workaround, google `nunit ignore attribute site:github.com`. I get the proper page as the first item in the list.

@ChrisMaddock There is no final decision that we will have a website on GitHub.There has been discussion, but nothing is final.

My view is that having docs on a wiki gives us a lot of benefit, so I wouldn't want to move the active docs to the website anyway. You may remember that they were always on the website before, until we decided to move them to the wiki.

I think this problem has two underlying causes:
1. It's hard to search GitHub wikis. There are several browser extensions that try to deal with this.
2. The NUnit practice of keeping old docs at the same location over the years means that the oldest documentation has had the largest number of hits and references.

By the way. It was the first time I ever did a google search for something in the docs. It was an education and I'm glad you brought it up.

I'm leaving this open as something to think about.

**Comment by dicko2** [comment]
I too have found this frustrating.

@CharliePoole what about using github pages (https://pages.github.com/) for hosting the site? these seem to be the common place among open source projects.

The best thing to sort out the google references is to put up 301 redirects form the old pages to the corresponding new ones.

e.g.
http://nunit.org/index.php?p=category&r=2.6.3 -> https://github.com/nunit/docs/wiki/Category-Attribute

But do you want to maintain the old pre-3.0 doco? if so I would move it to a new URL such as 

http://nunit.org/arc/

So that it can still be accessed, but all the high ranking google search results will start to go to 3.0+ doco on the new site and this should bleed the page ranking over to the new site.

Just my 2 cents :smile:

**Comment by CharliePoole** [comment]
So the old and new docs are two different issues in my mind.

The old docs are on the website we have had for years. The pages are html fragments and since we maintain docs for every past version of NUnit there are at least an order of magnitude more pages than for the new docs. The plan is to keep these for archival purposes without converting them in any way, no matter where the website is eventually hosted. These docs will become less important as time goes on.

For 3.0, we decided to create a wiki instead. It was under the nunit  repo at first but has now been moved to it's own docs  repo, as you know. The pages were created for the most part by hand editing the old 2.6.4 docs, changing the html to markdown, and updating as needed. We continue to improve the docs and every contributor is able to edit it. I don't see any benefit in putting these back into a web site, since that's what we just migrated away from.

I don't know how people are generally getting to the docs. If they start from GitHub, they should end up right at the wiki. If they start from the website, they have to click a few times to get there, which is probably not as good as it should be, but seems workable.

So for me, the question is how to make the current docs more searchable without moving them. One way would be to move all the older docs to a separate page as you suggest. We probably need to keep 2.6.4 on the front page of the docs for a while, since it is still our stable V2 release and has different docs from 3.0.
